index.js

```diff
@@ -14,9 +14,10 @@ class CloudinaryAdapter extends BaseAdapter{
 
   exists(filename) {
     return new BlueBird(function(resolve) {
-      cloudinary.v2.api.resource(path.parse(filename).name, {type: 'upload'}, function(error, result) {
+      // Use explicit instead of resource because there's no rate limit for explicit
+      cloudinary.v2.uploader.explicit(path.parse(filename).name, {type: 'upload'}, function(error, result) {
         if (result) {
-          resolve(result);
+          resolve(true);
```

Comment:
We don't need the result here (for ghost)? With only the true  resolve are we able to link the image to the post?

Reply:
The image URL in the post is returned by the `save` method. The `exists` method only returns a boolean.
However, I could just return the boolean without the resolve.